### DISCUSSION TITLE
Problem running setup.sh with  "yarn install"

### DISCUSSION THREAD

**Discussion by supernova1963** [ej52/proxmox-scripts#127] [Q&A]
What am I doing wrong?

> #npm:~# cat /etc/os-release
> NAME="Alpine Linux", ID=alpine
> VERSION_ID=3.18.0
> PRETTY_NAME="Alpine Linux v3.18"
> HOME_URL="https://alpinelinux.org/",BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"

> npm:~# python --version
> Python 3.11.4

> [error] Building frontend
> /tmp/npm_install.sh: line 180: yarn install
> 205: yarn install
> [] 942/942warning " > marionette.templatecache@1.0.0" has incorrect peer dependency "backbone@~1.3.3".
> " > marionette.templatecache@1.0.0" has incorrect peer dependency "underscore@~1.8.3".
> "marionette.templatecache > backbone.marionette@4.0.0-beta.1" has incorrect peer dependency "backbone@~1.3.3".
> "marionette.templatecache > backbone.marionette@4.0.0-beta.1" has incorrect peer dependency "underscore@>= 1.8.3 <= 1.9.x".
> "tabler-ui > bootstrap@4.5.1" has unmet peer dependency "popper.js@^1.16.1".
> Command failed.

[setup_sh.log](https://github.com/ej52/proxmox-scripts/files/12216178/setup_sh.log)

**Comment by ej52** [comment]
@supernova1963  Please try running the latest install script, as it should have fixed any issues.
